DuplicatesFinder_v4/ViewModels/DuplicatesViewModel.cs

```diff
@@ -71,7 +71,7 @@ public async void SaveToTxt()
         {
             var dateCreation = DateTime.Now.ToShortDateString();
             var setFileSaveName = $"Dublicates_{dateCreation}.txt";
-            var setFilePath = Path.Combine($"{PathWithAppFolder}", "{setFileSaveName}");
+            var setFilePath = Path.Combine($"{PathWithAppFolder}", $"{setFileSaveName}");

```

Comment:
just PathWithAppFolder, no need in forrmatting

Reply:
Fixed. Tnx